replace_triggered_by docs (#30931)
hashicorp/terraform: website/docs/language/meta-arguments/lifecycle.mdx

  used in the expression to index specific instances.

  `replace_triggered_by` is used to couple the lifecycles of otherwise
  independent resources, allowing the configurable replacement of a resource
  based on conditions in a dependency. Given the three classes of references
  allowed in `replace_triggered_by`, the following conditions can apply:
  - If the reference is to a resource with multiple instances, a plan to
    update or replace any instance will trigger replacement.
  - If the reference is to a single resource instance, a plan to update or
    replace that instance will trigger replacement.
  - If the reference is to a single attribute of a resource instance, any
    difference in the before and after value will trigger replacement.

  Only managed resources are allowed in the `replace_triggered_by` arguments.
  Because the `replace_triggered_by` expressions are used to lookup changes in
  managed resources, the `replace_triggered_by` argument can be modified
  without forcing replacement.

  ```hcl
  resource "aws_appautoscaling_target" "ecs_target" {
    # ...
    lifecycle {
      replace_triggered_by = [

>         # This resource must be replaced each time the aws_ecs_service is replaced.
>         # If we don't want to trigger on every change, we can use a computed
>         # attribute like id to trigger only on instance replacement.

Reviewer: I want to try to make the comment shorter and more concise. Do I lose anything here? 
```suggestion
        # Replace `aws_appautoscaling_target` each time this instance of 
        # the `aws_ecs_service` is replaced.
```
